devel/tapi/Portfile

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+
+name                tapi
+version             1000.10.8
+revision            0
+categories          devel
+platforms           darwin
+maintainers         {kencu @kencu} openmaintainer
+
+license             APSL-2
+description         ${name} adds ld64 linker support for text-based API libraries.
+long_description    ${description} Newer MacOS SDKs contain only these TAPI stubs.
+
+homepage            http://opensource.apple.com/source/${name}/
+master_sites        http://opensource.apple.com/tarballs/${name}:tapi
+
+distfiles           ${name}-${version}.tar.gz:tapi
```

Comment:
I don't think this redundant line is necessary. Just name the distfile in the `checksums` line below.

Reply:
doesn't seem to work like that for me. Just fails.